tests/opentracer/test_scope.py

```diff
@@ -1,9 +1,10 @@
+import pytest
+
 from ddtrace.opentracer.scope import Scope
 from ddtrace.opentracer.scope_manager import ScopeManager
 from ddtrace.opentracer.span import Span
 
-
-class TestScope(object):
-    def test_init(self):
-        scope = Scope(ScopeManager(), Span(None, None, None))
-        assert scope is not None
+# class TestScope(object):
+#     def test_init(self):
+#         scope = Scope(ScopeManager(), Span(None, None, None))
+#         assert scope is not None
```

Comment:
If this is unused, we should just delete it. No sense keeping dead code around in comments.